iohub/ngff/nodes.py

```diff
@@ -310,20 +310,24 @@ def close(self):
 class ImageArray(zarr.Array):
     """Container object for image stored as a zarr array (up to 5D)"""
 
-    def __init__(self, zarray: zarr.Array):
-        super().__init__(
-            store=zarray._store,
-            path=zarray._path,
-            read_only=zarray._read_only,
-            chunk_store=zarray._chunk_store,
-            synchronizer=zarray._synchronizer,
-            cache_metadata=zarray._cache_metadata,
-            cache_attrs=zarray._attrs.cache,
-            partial_decompress=zarray._partial_decompress,
-            write_empty_chunks=zarray._write_empty_chunks,
-            zarr_version=zarray._version,
-            meta_array=zarray._meta_array,
-        )
+    def __init__(self, zarray: zarr.Array = None, **kwargs):
+        if zarray is not None:
```

Comment:
Can you explain when this method should be called without a argument? It is an internal wrapper, and if you don't need the wrapper, just doing `Position.zgroup["0"]` would give you the plain `zarr.Array` object.

Reply:
I added an example use case at the top and a short docstring to the `__init__` function:

```python
    def __init__(self, zarray: zarr.Array = None, **kwargs):
        """Keyword arguments are passed to the zarr.Array constructor.
        If a zarr.Array is provided, the constructor will use its attributes
        to initialize the ImageArray.
        """
```

I think this is a common enough use case, such that we should make iohub arrays work with the dask arrays in multiprocessing applications